apm-agent-plugins/apm-micrometer-plugin/src/test/java/co/elastic/apm/agent/micrometer/MicrometerMetricsReporterTest.java

```diff
@@ -140,6 +142,7 @@ void testDisableDedotMetricName() {
     }
 
     @Test
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
Why? What's not working with this test?

Reply:
fixed with adding charset: `.map(k -> new String(k, StandardCharsets.UTF_8))`